.github/workflows/rust.yml

```diff
@@ -0,0 +1,71 @@
+# Based on https://github.com/actions-rs/meta/blob/master/recipes/quickstart.md
+
+on: [push, pull_request]
+
+name: Rust
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
```

Comment:
I think having latest means less reproducibility. It would be good to use a tagged version.

Reply:
Fixed, just fixed it to ubuntu-18.04